UITests/Global.swift

```diff
@@ -195,7 +195,7 @@ extension KIFUITestActor {
 
         webView.evaluateJavaScript("typeof KIFHelper") { result, _ in
             if result as! String == "undefined" {
-                let bundle = Bundle(for: NavigationTests.self)
```

Comment:
After removing NavigationTets.swift (the UITests) there was a compilation error here in Global.swift. Is it possible to replace it by antoher class in the same repo? In terms of the tests it does not affect, it was changed by BrowserTests and the build compiled and tests passed, but I do not really understand the purpose of its use here and how it works so maybe I am missing something. Could you please help me understand how it works and whether we can replace it? @sleroux or @thebnich, Thanks!!